e2e/rdctl.e2e.spec.ts

```diff
@@ -834,6 +835,23 @@ test.describe('Command server', () => {
         }
       });
     });
+    test('should list images based on current containerEngine', async() => {
+      const { stdout, stderr, error } = await rdctl(['list-settings']);
+      const settings = JSON.parse(stdout);
+      const containerEngine = settings.kubernetes.containerEngine;
+
+      expect(containerEngine).toMatch(/containerd|moby/);
```

Comment:
```suggestion
      expect(settings).toHaveProperty('kubernetes.containerEngine', expect.StringMatching(/containerd|moby/));
```
It's basically the same test, but spews out the whole settings object if it fails.

Reply:
Do we want `/^(?:containerd|moby)$/` or something complicated like that?

Reply:
Went with the complete-name match.